test/integration/tests.js

```diff
@@ -379,6 +394,22 @@ test('Gets entries by creation order and id order', async (t) => {
   )
 })
 
+test.only('Gets an entry that has resource links', async (t) => {
+  t.plan(8)
+  const response = await client.getEntry('6yfSzwXo99q8BKzkE5AIKo')
```

Comment:
is this a real entry id that we have in the test space? I wonder if we could have a fixed id (e.g. `crossSpaceEntry`) instead.

Reply:
Yes it's a real entityId. Do you know if it's possible to change this id? If so we can do that.

![Screenshot 2022-09-22 at 10 30 27](https://user-images.githubusercontent.com/11991830/191698254-ca2e349f-02d8-4416-bec6-55bea149a6d1.png)

Reply:
You can create entries with your own IDs through the CMA API  (`POST /entries/myEntryId` or something like it) -- unfortunately not trough the webapp afaik